.github/workflows/style.yml

```diff
@@ -35,12 +35,12 @@ jobs:
 
     - name: Run isort
       run: |
-        isort --check komodo tests *.py
+        isort --check komodo tests
 
     - name: Lint with flake8
       run: |
-        flake8
+        flake8 --max-line-length 88
```

Comment:
Just a reminder for what was discussed: Shall we instead make it a pylint check for the max-line-length and source the value from `pyproject.toml`?

Reply:
See @berland's comment above: `flake8` needs the constraint otherwise it will fail.